server.js

```diff
@@ -1,10 +1,23 @@
 const express = require('express')
 const app = express()
+const mongoose = require('mongoose')
+const URI = 'mongodb+srv://Rishabh:qwertyuiop@cluster0.8yyadat.mongodb.net/?retryWrites=true&w=majority'
```

Comment:
Consider using environment variables for sensitive data like MongoDB URI. This will make your application more secure and configurable. You can use packages like 'dotenv' to load environment variables from a .env file. [important]